src/utils/decorators/input-validation/input-validation.js

```diff
@@ -633,9 +625,22 @@ const InputValidation = (ComposedComponent) => {
         iconStyle = { [`${this.props.align}`]: `${100 - this.props.labelWidth}%` };
       }
 
-      if (this.state.messageLocked) { messageClasses += ' common-input__message--locked'; }
```

Comment:
do we not need this class any more?

Reply:
I provoked the question with the rename. I just renamed it to ```common-input__message--shown``` because I could see no reason why it shouldn't be in the same css state if it is shown or locked.